src/components/Map.vue

```diff
@@ -264,9 +265,7 @@ export default {
         this.clearAvailableRoutes();
 
         this.detectives.map(detective => {
-          const point = document.querySelector(
-            `[data-point="${detective.currentLocation.toString()}"]`
-          );
+          const point = this.$refs[detective.currentLocation][0];
```

Comment:
Bro can you explain that this line of code is doing? From what I know, $refs are used to access elements of the DOM that have a ref attribute in their tags right? eg: <div ref="mydiv">

$ref.mydiv would access the above div

Reply:
Since detective is passed in to the component as a prop, is there a reason why you wouldnt access the current location directly? as detective.currentLocation[0] ?

Reply:
Since I am creating a `ref` for each station by it's station number in the Map.vue component - https://github.com/azharkhan/scotland-yard/blob/master/src/components/Map.vue#L17

I can basically do the same thing I did above with the querySelector, but use `refs` for it. 

However, a small gotcha with setting refs in a `v-for` loop is that the ref is set as an Array - which is why I need to access the first index, even though there's only one ref in the Array for each station.

See Also: https://vuejs.org/v2/api/#ref

> When used on elements/components with v-for, the registered reference will be an Array containing DOM nodes or component instances.